.github/workflows/publish.yml

```diff
@@ -1,70 +1,61 @@
-name: Publish Packages
+name: publish
 on:
-  repository_dispatch:
-    types: [semantic-release]
+  pull_request:
+    types: [closed]
 jobs:
   publish_packages:
-    runs-on: 'ubuntu-latest'
+    if: github.event.action == 'closed' && github.event.pull_request.merged
+    runs-on: ubuntu-latest
     steps:
-      - name: 'clone repo with full history for lerna'
+      - name: check out latest commit
         uses: actions/checkout@v2
         with:
-          fetch-depth: 0
```

Comment:
Is `lerna` able to version without this setting?

Reply:
Can we do an example run with `on: pull_request(synchronize)`?

Reply:
Beats me. I know nothing about `lerna`. I _do_ know that packages are published when this workflow is run. I _don't_ know if that one example is sufficiently comprehensive.

Reply:
Regarding a test run: I commented out the `on` criteria and the `if` condition on the `publish_packages` job, and [packages published as expected](https://github.com/rentpath/react-ui/packages).

Reply:
Awesome